posts/why-is-react-native-so-promising.md

```diff
@@ -0,0 +1,161 @@
+---
+authors:
+- Valentin Carmignac
+categories:
+- Javascript
+- React Native
+date: 2018-02-22T13:47:51.000Z
+title: "Why's React Native so promising?"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2018/02/react-native-02-2018-thumbnail.png
+---
+
+This year with Ippon in Melbourne, Australia we've had the chance to build an iPad application. Being the designated front-end developer in the agency, I got to choose the stack for this project. I'm not very big on native technologies but I do enjoy a fair bit of React; that's why I was excited to try out React Native. 
+
+Now this article is not an introduction to React. This is merely a gathering of the thoughts I had and discoveries I made during this project. React Native is very good and hopefully this article will bring more people to use it and help bring the framework to maturation.
+
+## React is **not** a framework. React Native is one.
+
+I find that in the frontend world, lots of people compare Angular and React as if they were equals. But there's an important distinction, React is only a rendering library. Whereas Angular bundles a lot of features together, you will need to use a lot of different libraries to have equivalent functionality with React. Redux being the most popular state management library to use with React you'll ship them together, making the stack a kind of framework. I think it is important to know the difference and understand that you cannot really compare them.
+
+Enter React Native. React Native looks a lot like React: roughly the same syntax, classes, JSX, state, props...but React Native is a framework, which means you have more features than just rendering in React Native. And that's simply because React Native needs to interact with the mobile it's running on. To do that, the Facebook team created a bunch of APIs to interact with the camera, the keyboard, the vibrations...
```

Comment:
Add a link to React Native's website the first time the framework is mentioned

Reply:
🚵